test/system/src/test/java/io/pravega/test/system/MultiControllerTest.java

```diff
@@ -46,7 +46,9 @@
     private static final ScheduledExecutorService EXECUTOR_SERVICE = Executors.newSingleThreadScheduledExecutor();
 
     private Service controllerService1 = null;
-    private ControllerImpl controllerClient = null;
+    private Service segmentStoreService = null;
+    private ControllerImpl controllerClientDirect = null;
```

Comment:
Why is this not using an atomic reference like the URI counterpart?

Reply:
using it now.